src/flowchart.symbol.input.js

```diff
@@ -0,0 +1,67 @@
+var Symbol = require('./flowchart.symbol');
+var inherits = require('./flowchart.helpers').inherits;
+var drawAPI = require('./flowchart.functions');
+var drawPath = drawAPI.drawPath;
+
+function Input(chart, options) {
+  options = options || {};
+  Symbol.call(this, chart, options);
+  this.textMargin = this.getAttr('text-margin');
+
+  this.text.attr({
+    x: this.textMargin * 3
+  });
+
+  var width = this.text.getBBox().width + 4 * this.textMargin;
+  var height = this.text.getBBox().height + 2 * this.textMargin;
+  var startX = this.textMargin;
+  var startY = height/2;
+
+  var start = {x: startX, y: startY};
+  var points = [
```

Comment:
The points seems to be the same as the output.js...
I think that should be changed to:

```js
var points = [
    {x: startX - this.textMargin + 2 * this.textMargin, y: height},
    {x: startX - this.textMargin + width, y: height},
    {x: startX - this.textMargin + width + 2 * this.textMargin, y: 0},
    {x: startX - this.textMargin, y: 0},
    {x: startX, y: startY}
  ];
```

Reply:
my local file like this:
`var points = [
    {x: startX - this.textMargin + 2 * this.textMargin, y: height},
    {x: startX - this.textMargin + width, y: height},
    {x: startX - this.textMargin + width + 2 * this.textMargin, y: 0},
    {x: startX - this.textMargin, y: 0},
    {x: startX, y: startY}
  ];
`

Reply:
now input.js is pushed. please check. Thank you!